### ISSUE TITLE
Update to Node 20

### ISSUE THREAD

**Issue by merlinpaypal** [yutailang0119/action-ktlint#836]
When running this action, I get this warning

```
Node.js 16 actions are deprecated. Please update the following actions to use Node.js 20: yutailang0119/action-ktlint@v3.
```

**Comment by yutailang0119** [comment]
@merlinpaypal I have released [v4.0.0](https://github.com/yutailang0119/action-ktlint/releases/tag/v4.0.0), which has been updated to Node 20.
Please give it a try.
